test/WebSites/MvcTagHelpersWebSite/Components/ProductsViewComponent.cs

```diff
@@ -16,7 +16,11 @@ public IViewComponentResult Invoke(string category)
         {
             IExpirationTrigger trigger;
             var products = ProductsService.GetProducts(category, out trigger);
-            EntryLinkHelpers.ContextLink.AddExpirationTriggers(new[] { trigger });
+
+            if (EntryLinkHelpers.ContextLink != null)
```

Comment:
It's designed to let you link two cache entries, not add triggers from arbitrary call sites. Don't bring it back without taking this up with @lodejard